pandas/core/common.py

```diff
@@ -2072,7 +2072,7 @@ def _random_state(state=None):
     elif isinstance(state, np.random.RandomState):
         return state
     elif state is None:
-        return np.random.RandomState()
+        return np.random.mtrand._rand
```

Comment:
I don't think that will do the same?

As this is to catch a previously called `np.random.seed()`, so feeding it another seed will not give the desired result I think?